packages/flutter/test/material/bottom_sheet_test.dart

```diff
@@ -2263,6 +2263,99 @@ void main() {
           tester.widget(find.byType(ModalBarrier).last);
       expect(modalBarrier.semanticsLabel, MaterialLocalizations.of(scaffoldKey.currentContext!).scrimLabel);
     });
+
+    testWidgetsWithLeakTracking('Swiping up an almost contracted sheet should expand it', (WidgetTester tester) async {
+
+      Future<void> flingFrom(
```

Comment:
this function is copied from source. the initial offset is useful to simulate a drag before the fling, but a bug in the implementation makes the initialoffset not functional. It needs to do at least 2 move events: first to detect a drag, second to actually drag. 

Smelly code duplication, need advice on how to handle this